README.md

```diff
@@ -125,8 +140,6 @@ Unless you have set up a reverse proxy to handle encrypted connections, use
 whatever you have specified in `ankisyncd.conf` (or, if using a reverse proxy,
 whatever port you configured to accept the front-end connection).
 
-**Do not use trailing slashes.**
```

Comment:
Just confirming you've fixed this?

Reply:
Yes, I have fixed this with ankidroid/Anki-Android#6437